src/Sarif.UnitTests/TestData/InsertOptionalDataVisitor/CoreTests.sarif

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
+  "runs": [
+    {
+      "tool": { "name": "Sarif.UnitTests",
+        "fullName": "Sarif.UnitTests.1.3.3.0-beta", "version": "1.3.3.0", "semanticVersion": "1.3.3", "sarifLoggerVersion": "1.5.22.0", "language": "en-US"},
```

Comment:
>1.5.22.0 [](start = 125, length = 8)

Is this _still_ the logger version?

Reply:
no. but i don't want to set a standard to update this information every time the logger version changes. 

---
In reply to: [205980807](https://github.com/Microsoft/sarif-sdk/pull/974#discussion_r205980807) [](ancestors = 205980807)